src/qililab/instruments/qblox/qblox_module.py

```diff
@@ -313,9 +312,7 @@ def setup(  # pylint: disable=too-many-branches, too-many-return-statements
             return
 
         if channel_id is None:
-            if port_id is not None:
-                channel_id = self.get_sequencers_from_chip_port_id(chip_port_id=port_id)[0].identifier
-            elif self.num_sequencers == 1:
+            if self.num_sequencers == 1:
```

Comment:
same issue as with the d5a, if we set a value for this, how do we re-set the value so that it is correct after instrument initialization?